packages/field-base/src/input-constraints-mixin.js

```diff
@@ -5,10 +5,20 @@
  */
 import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
 import { InputMixin } from './input-mixin.js';
-import { ValidateMixin } from './validate-mixin.js';
+import { DelegateInputStateMixin } from './delegate-input-state-mixin.js';
 
 const InputConstraintsMixinImplementation = (superclass) =>
-  class InputConstraintsMixinClass extends ValidateMixin(InputMixin(superclass)) {
+  class InputConstraintsMixinClass extends DelegateInputStateMixin(InputMixin(superclass)) {
```

Comment:
We probably don't need `InputMixin` here, because `DelegateInputStateMixin` applies it.